pages/glossary/essential-text-change.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Essential Text Change
+key: essential-text-change
+unambiguous: true
+objective: false
+input_aspects:
+  - Accessibility tree
+  - CSS styling
+  - DOM tree
+---
+
+An _essential text change_ occurs when the modification of the text content of an HTML element is necessary to convey accurate and timely information to the user. An _essential text change_ may appear in any of the following forms:
+
+1. Providing context - Text is changed to provide users with additional information required to understand the current state of the page. For example, while searching for results, text may change to convey that the page is *loading* to provide context to the user.
+2. Live updates of important information - The content automatically updates to show the most recent information available from a data source and should not be ignored by the user. Importance of information is often subjective and requires some amount of human understanding, so we have provided examples below to help illustrate when text should be considered important.
```

Comment:
This is still open ended. What I was really hoping for here is a semi-definitive list of things that are essential. Anything that isn't on the list isn't essential. I tried to do something like that with the definition of [essential text presentation](https://github.com/act-rules/act-rules.github.io/pull/1969/files#diff-4a01a104eef4098b17b94e7fdf93e7720a0da132098bd10a77e8624e9a0081c3R2).

Reply:
Agree that It is open-ended, but this is not an objective definition. Does it really have to be close-ended?